disco.py

```diff
@@ -379,48 +434,70 @@ def createPath(filepath):
     root_path = f'/content'
     model_path = '/content/models' 
 else:
-  root_path = f'.'
-  model_path = f'{root_path}/model'
+  root_path = os.getcwd()
+  model_path = f'{root_path}/models'
 
 model_256_downloaded = False
 model_512_downloaded = False
 model_secondary_downloaded = False
 
+multipip_res = subprocess.run(['pip', 'install', 'lpips', 'datetime', 'timm', 'ftfy', 'einops', 'pytorch-lightning', 'omegaconf'], stdout=subprocess.PIPE).stdout.decode('utf-8')
+print(multipip_res)
+
 if is_colab:
-  gitclone("https://github.com/openai/CLIP")
-  #gitclone("https://github.com/facebookresearch/SLIP.git")
-  gitclone("https://github.com/crowsonkb/guided-diffusion")
-  gitclone("https://github.com/assafshocher/ResizeRight.git")
-  gitclone("https://github.com/MSFTserver/pytorch3d-lite.git")
-  pipie("./CLIP")
-  pipie("./guided-diffusion")
-  multipip_res = subprocess.run(['pip', 'install', 'lpips', 'datetime', 'timm', 'ftfy'], stdout=subprocess.PIPE).stdout.decode('utf-8')
-  print(multipip_res)
   subprocess.run(['apt', 'install', 'imagemagick'], stdout=subprocess.PIPE).stdout.decode('utf-8')
-  gitclone("https://github.com/isl-org/MiDaS.git")
-  gitclone("https://github.com/alembics/disco-diffusion.git")
-  pipi("pytorch-lightning")
-  pipi("omegaconf")
-  pipi("einops")
-  # Rename a file to avoid a name conflict..
-  try:
-    os.rename("MiDaS/utils.py", "MiDaS/midas_utils.py")
-    shutil.copyfile("disco-diffusion/disco_xform_utils.py", "disco_xform_utils.py")
-  except:
-    pass
 
-if not os.path.exists(f'{model_path}'):
-  pathlib.Path(model_path).mkdir(parents=True, exist_ok=True)
-if not os.path.exists(f'{model_path}/dpt_large-midas-2f21e586.pt'):
-  wget("https://github.com/intel-isl/DPT/releases/download/1_0/dpt_large-midas-2f21e586.pt", model_path)
+try:
```

Comment:
It's really cool that these installation steps pull everything without polluting the Python env. That sidesteps much of my concern about automating installation in a local run.